ores/wsgi/routes/v2.py

```diff
@@ -0,0 +1,178 @@
+            return responses.bad_request("Model '{0}' not available for {1}."
+                                         .format(model, context))
+
+        # Try to read the rev_ids
+        if 'revids' in request.args:
+            try:
+                rev_ids = set(read_bar_split_param(request, "revids", type=int))
+            except ParamError as e:
+                return responses.bad_request(str(e))
+
+            if len(rev_ids) == 0:
+                return responses.bad_request("No revids provided.")
+        else:
+            return jsonify(score_processor[context][model].info())
+
+        # Get model info
+        try:
+            model_info_req = set(read_bar_split_param(request, "model_info", type=str))
+        except ParamError as e:
+            return responses.bad_request(str(e))
+
+        precache = "precache" in request.args
+        try:
+            model_object = score_processor[context][model]
+            model_info = {model: {'version': model_object.version}}
+            if model_info_req:
+                for req in model_info_req:
+                    try:
+                        model_info[model][req] = model_object.info()[req]
+                    except KeyError:
+                        return responses.bad_request(
+                            "Model '{0}' has not attribute {1}.".format(
+                                model, req))
+            scores = {model: score_processor.score(context, model, rev_ids,
+                                                   precache=precache)}
+        except errors.ScoreProcessorOverloaded:
+            return responses.server_overloaded()
+        return format_output(context, scores, model_info)
+
+    # /v2/scores/enwiki/reverted/4567890
+    @bp.route("/v2/scores/<context>/<model>/<int:rev_id>/", methods=["GET"])
+    def score_revision_v2(context, model, rev_id):
+
+        # Check to see if we have the context available in our score_processor
+        if context not in score_processor:
+            return responses.not_found("No scorers available for {0}"
+                                       .format(context))
+
+        precache = "precache" in request.args
+        model_object = score_processor[context][model]
+        model_info = {model: {'version': model_object.version}}
+        scores = {}
+        # If the model exists, score revisions with it and return the result
+        if model not in score_processor[context]:
+            return responses.not_found("Model '{0}' not available for {1}."
+                                       .format(model, context))
+
+        # Get model info
+        try:
+            model_info_req = set(read_bar_split_param(
+                request, "model_info", type=str))
+        except ParamError as e:
+            return responses.bad_request(str(e))
+        if model_info_req:
+            for req in model_info_req:
+                try:
+                    model_info[model][req] = model_object.info()[req]
```

Comment:
This will potentially be problematic. 

So, model_info.roc.fpr, model_info.roc.tpr, and model_info.roc.thresholds contains a set of values for plotting the actual ROC curve that a user would usually not want.  We need to find some way to make sure that we just return model_info.roc.auc by default when "roc" appears in "model_info" as well as some way to request all of the values for plotting when necessary.